cli/flox/src/utils/errors.rs

```diff
@@ -515,6 +515,34 @@ pub fn format_locked_manifest_error(err: &LockedManifestError) -> String {
         // already user friendly.
         // some commands catch these errors and process them separately
         // e.g. `flox pull`, `flox push`
+
+        // 116: toml parsing error, just print the error message
+        // https://github.com/flox/flox/issues/852
+        LockedManifestError::LockManifest(CallPkgDbError::PkgDbError(PkgDbError {
+            exit_code: 116,
+            context_message:
+                Some(ContextMsgError {
+                    caught: Some(caught),
+                    ..
+                }),
+            ..
+        })) => {
+            let message = &caught.message;
+            let un_prefixed = message.strip_prefix("[error] ").unwrap_or(message);
+            formatdoc! {"
+                {un_prefixed}
+            "}
+        },
+        // 105: invalid lockfile, just print the error message
+        // https://github.com/flox/flox/issues/852
+        LockedManifestError::LockManifest(CallPkgDbError::PkgDbError(PkgDbError {
+            exit_code: 105,
+            context_message: Some(ContextMsgError { message, .. }),
+            ..
+        })) => formatdoc! {"
+            {message}
+        "},
```

Comment:
should these errors _always_ be unpacked, or is this an edit only optimization.
I'm hoping for the prior, as well as a more categorical solution to this :)

Reply:
I think this is just a case where we know we specifically want the `context_message` field. Since we're passing structured errors from the C++ side it's possible that in one place the `caught_message` might be the useful bit (e.g. some error about a particular manifest field) and in another place the `caught_message` part may be too low-level and instead we want the `context_message` field.